link-grammar/build-disjuncts.c

```diff
@@ -347,6 +347,41 @@ void prt_exp(Exp *e, int i)
 		printf("con=%s\n", e->u.string);
 	}
 }
+
+void prt_exp_mem(Exp *e, int i)
```

Comment:
I believe that there already is a print function that is more-or-less the same as this, somewhere else in the code...